docs/introduction.rst

```diff
@@ -0,0 +1,40 @@
+Introduction
+============
+
+Trinity & Py-EVM
+~~~~~~~~~~~~~~~~
+
+Py-EVM is a new implementation of the Ethereum Virtual Machine (EVM) written in Python. Trinity is the client software that connects to the Ethereum network and runs on top of Py-EVM.
+
+Trinity and Py-EVM aim to replace existing Python Ethereum implementations to eventually become the defacto standard for the Python ecosystem.
+
+If none of this makes sense to you yet we recommend to checkout the `Ethereum <https://ethereum.org>`_ website as well as a `higher level description <http://www.ethdocs.org/en/latest/introduction/what-is-ethereum.html>`_ of the Ethereum project.
+
+Goals
+-----
+
+The main focus is to enrich the Ethereum ecosystem with a Python implementation that is:
+
+* Well documented
+* Easy to understand
+* Has clear APIs
+* Runs fast and resource friendly
+* Is highly flexible to support:
+
+  * Public chains
+  * Private chains
+  * Consortium chains
+  * Advanced research
+
+.. note::
+
+  This project is still in an alpha state and not yet meant to be used by regular consumers to connect to the main Ethereum network. We are quickly progressing forward and shoot for an MVP in a matter of weeks rather than months.
```

Comment:
@pipermerriam please review if this aligns with what you think the current state and timeline is.

Reply:
I'm a little pessimistic about weeks vs months, but... that's just my nature. :) I think we can be optimistic here, with little ramifications.